transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringSubmissionQueue.java

```diff
@@ -28,6 +28,11 @@
 final class IOUringSubmissionQueue {
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(IOUringSubmissionQueue.class);
 
+    // this can be set smaller than the ring size to flush submissions more frequently
+    private static final int MAX_SUBMISSION_BATCH_SIZE = Integer.MAX_VALUE;
```

Comment:
it's probably not good idea to use as default, if events are constantly coming in cqe loop(for example 200 etc., it will "never" be submitted, latency would be very bad